task3_dictionary_practice.py

```diff
@@ -0,0 +1,34 @@
+"""
+Task2.
+
+Usages of dictionary comprehensions.
+"""
+
+
+def generate_numbers(n=20):
+    """Return a dictionary with square if keys"""
+    import math
+    dic = {}
+    for x in range(n):
+        dic[x + 1] = int(math.pow(x + 1, 2))
+    return dic
+
+
+if __name__ == '__main__':
```

Comment:
We usually don't use if __name__ = '__main__':
twice. We use it for testing in the end of the module